package.json

```diff
@@ -1,38 +1,24 @@
 {
-  "name": "cesium-webpack-example",
+  "name": "cesiumjs-webpack-tutorial",
   "version": "1.0.0",
-  "description": "The minimal recomended setup for an app using Cesium with Webpack.",
+  "description": "Getting started with CesiumJS and Webpack.",
   "main": "index.js",
-  "keywords": [
-    "cesium",
-    "webpack",
-    "example"
-  ],
-  "author": {
-    "name": "Cesium GS, Inc.",
-    "url": "https://cesium.com"
-  },
-  "homepage": "https://cesiumjs.org",
-  "license": "Apache-2.0",
-  "dependencies": {
-    "cesium": "^1.63.1"
+  "scripts": {
+    "build": "node_modules/.bin/webpack --config webpack.config.js",
+    "start": "node_modules/.bin/webpack serve --config webpack.config.js --open"
   },
+  "author": "Sam Rothstein",
```

Comment:
Make sure author/keyword/license information is all reverted back to the previous information.  Also update the urls so they point to cesium.com

Reply:
Not sure why this comment was resolved, but please don't forget about the above.